packages/plugin-auto-tracking-browser/src/auto-tracking-plugin.ts

```diff
@@ -15,12 +18,21 @@ interface EventListener {
 }
 
 interface Options {
+  tagAllowlist?: AllowedTag[];
   cssSelectorAllowlist?: string[];
```

Comment:
Can you please also add descriptions of these settings here so they appear in intellisense?